platform-sdk/docs/core/platform-status.md

```diff
@@ -1,70 +0,0 @@
-# Platform Status
-
-The platform status is represented as an enum value from `PlatformStatus.java`. It is updated via
```

Comment:
Was deleting this file intentional?

Reply:
I expected this to be updated to explain the new paradigm

Reply:
@poulok This deletion was intentional.

I started describing the new paradigm in text form, and found the result unhelpful. It was just a less digestible version of the state machine diagram, which already includes all relevant information regarding what events cause which transitions.

It also doesn't seem helpful to include documentation describing the characteristics of each platform status, as that documentation already exists in the `PlatformStatus` enum itself.

Reply:
I think there is still benefit to a textual supplement to the diagram. For example, why do we need a CHECKING state? Why is it necessary for the reconnect state to be saved to disk prior to moving to the CHECKING state? What is the purpose of the OBSERVING state? The "what" is explained well in the code, but not the why. Those questions are the more difficult to answer, so having a document with more context in it is useful.